book/src/contributing/README.md

```diff
@@ -3,7 +3,7 @@
 If you're interested in joining the community to build a better SQL, here are
 ways to start:
 
-- Star this repo.
+- Star this repository.
```

Comment:
No need, keep informal

Reply:
I figured keeping it more formal would be the appropriate thing to do since it would give a more professional vibe.

Reply:
I think the "informal but informed" voice is best; "repository" is too formal